### PR TITLE
Add s3 Export description table

### PR DESCRIPTION
**Pull request by markzegarelli**
# Amplitude Developer Docs PR

## Description

Added a description table for the fields in the s3 export schema.

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [x] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [x] My documentation follows the style guidelines of this project.
- [x] I previewed my documentation on a local server using `mkdocs serve`.
- [x] Running `mkdocs serve` didn't generate any failures.
- [x] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/amazon-s3.md [hunk 1]
```diff
@@ -66,6 +66,61 @@ To disable automatic exports, open the integration and click **Manage**. You can
...
+| `$schema`                            | integer   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `adid`                               | string    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (joez2) [inline docs/data/destinations/amazon-s3.md:76]: Let's remove this as it's deprecated

File: docs/data/destinations/amazon-s3.md [hunk 2]
```diff
@@ -66,6 +66,61 @@ To disable automatic exports, open the integration and click **Manage**. You can
...
+| `$insert_id`                         | string    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
+| `$schema`                            | integer   |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (joez2) [inline docs/data/destinations/amazon-s3.md:75]: Deprecated: https://community.amplitude.com/announcements-56/deprecating-fields-from-the-export-data-format-1850

File: docs/data/destinations/amazon-s3.md [hunk 3]
```diff
@@ -66,6 +66,61 @@ To disable automatic exports, open the integration and click **Manage**. You can
...
+| `amplitude_attribution_ids`          | array     | Hashed attribution ids on the event                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `amplitude_event_type`               | string    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (joez2) [inline docs/data/destinations/amazon-s3.md:78]: Same

File: docs/data/destinations/amazon-s3.md [hunk 4]
```diff
@@ -66,6 +66,61 @@ To disable automatic exports, open the integration and click **Manage**. You can
...
+| `data`                               | dict      | Dictionary where certain fields such as `first_event` and `merged_amplitude_id` are stored                                                                                                                                                                                                                                                                                                                                                                                                                                                       |     |
+| `device_brand`                       | string    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (joez2) [inline docs/data/destinations/amazon-s3.md:86]: Removed as deprecated: https://community.amplitude.com/announcements-56/deprecating-fields-from-the-export-data-format-1850

File: docs/data/destinations/amazon-s3.md [hunk 5]
```diff
@@ -66,6 +66,61 @@ To disable automatic exports, open the integration and click **Manage**. You can
...
+| `device_id`                          | string    | The device specific identifier. Example: C8F9E604-F01A-4BD9-95C6-8E5357DF265D                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `device_manufacturer`                | string    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (joez2) [inline docs/data/destinations/amazon-s3.md:90]: Removed as deprecated: https://community.amplitude.com/announcements-56/deprecating-fields-from-the-export-data-format-1850

File: docs/data/destinations/amazon-s3.md [hunk 6]
```diff
@@ -66,6 +66,61 @@ To disable automatic exports, open the integration and click **Manage**. You can
...
+| `device_model`                       | string    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (joez2) [inline docs/data/destinations/amazon-s3.md:91]: Removed as deprecated: https://community.amplitude.com/announcements-56/deprecating-fields-from-the-export-data-format-1850

File: docs/data/destinations/amazon-s3.md [hunk 7]
```diff
@@ -66,6 +66,61 @@ To disable automatic exports, open the integration and click **Manage**. You can
...
+| `start_version`                      | string    | App version the user was first tracked on. Example: 1.0.0                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| `user_creation_time`                 | timestamp | `event_time` (UTC) of the user's first event. Example: `2015-08-10T12:00:00.000000`                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (joez2) [inline docs/data/destinations/amazon-s3.md:118]: Removed as deprecated: https://community.amplitude.com/announcements-56/deprecating-fields-from-the-export-data-format-1850

**Review by joez2 — APPROVED** [review]
LGTM! Thanks for making this change

**Review by zheniasemenina — APPROVED** [review]
Looks good! One q - @joez2 you marked a bunch of fields as 'deprecated', does it mean that our example of the exported JSON object has to be adjusted too? (line 136 onwards) 
Or will these fields still be exported but show as null/empty?

**Comment by joez2** [comment]
> Looks good! One q - @joez2 you marked a bunch of fields as 'deprecated', does it mean that our example of the exported JSON object has to be adjusted too? (line 136 onwards) Or will these fields still be exported but show as null/empty?

These fields will show as null, we keep them for backward compatibility reason, but we should remove them from documentation.

**Pull request merged**